src/rust/iced-x86/src/formatter/fast/trait_options_fast_fmt.rs

```diff
@@ -12,7 +12,6 @@ use crate::formatter::fast::trait_options::SpecializedFormatterTraitOptions;
 pub struct DefaultFastFormatterTraitOptions;
 
 impl SpecializedFormatterTraitOptions for DefaultFastFormatterTraitOptions {
-	#[doc(hidden)]
```

Comment:
This should not be removed.

Reply:
While running the tests cargo threw a message:
```
warning: `#[doc(hidden)]` is ignored on trait impl items
  --> iced-x86/src/formatter/fast/trait_options_fast_fmt.rs:15:2
   |
15 |     #[doc(hidden)]
   |     ^^^^^^^^^^^^^^ help: remove this attribute
   |
   = note: `#[warn(unused_attributes)]` on by default
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: whether the impl item is `doc(hidden)` or not entirely depends on the corresponding trait item
```
If this isn't relevant I can restore it

Reply:
Ok, seems like it wasn't allowed so let's keep your code then!